selfdrive/car/honda/carcontroller.py

```diff
@@ -154,8 +154,10 @@ def update(self, CC, CS):
       if self.frame % 10 == 0:
         can_sends.append((0x18DAB0F1, 0, b"\x02\x3E\x80\x00\x00\x00\x00\x00", 1))
 
+    lateral_active = CC.enabled if self.CP.minSteerSpeed > 0. else CC.latActive
```

Comment:
We should definitely still use latActive. Why do you want to keep the apply bit when there's a fault or we're below the min steer speed?

Reply:
Below the minSteerSpeed, sure latActive would be fine.

Marking the radar not accepting steer torque requests as a steering fault is kind of a misnomer on my part, but is how I elected to get this working without changes to the latActive policy in controlsd. As things are now, if latActive is true when we can't steer, that can max out the steering controller which would set Turn Exceeds Limit warnings. Also, when steering does become available, we wouldn't start the torque command at a baseline of 0 as the steer torque controller would already be active.

Did you have something else in mind?

Reply:
Can you not set the steer fault when this happens?

Reply:
Yeah the feedback loop that you describe is largely the issue at hand. I'll see what I can come up with to do away with a lot of these kinds of changes.

Reply:
Qt cabana has been really great so far and seems to be coming along nicely.

I spent a good chunk of time looking for a can signal and the camera does send a flag that's sort of related to a "steering available" signal, but it can go to zero even when the radar is allowing steering messages through. It appears to be more of a `ready` flag for the stock camera and its capabilities, so it's not a perfect indicator for what we're needing.

I'm putting together a branch to handle/clean-up the various minSteerSpeed/steer enable speeds and duplicate alert code which would apply to all the car brands (something along the lines of your idea here > https://github.com/commaai/cereal/commit/4e3fd007152f9cf46fbc2bd12ade132b33982700). I'll get it cleaned up and submitted as a separate pr.

Reply:
You'll want to be looking for a message from the radar.

Reply:
At `343` in `c443ca9eee127f66|2022-08-22--19-25-29`, speed drops below the `minSteerSpeed` and the radar drops our steer request as the car goes into that turn.

As for steer control beginning with OP already engaged, check out `464`. vEgo exceeds the steer engage threshold there. 

Bits 3 and 4 in Byte 4 of `0x33DB` on bus 2 are the closet I've ever found to what we're after; that's from the camera though. I haven't seen anything like it from the radar.